Classes/EventStore/EventStoreManager.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace Neos\EventSourcing\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\EventSourcing\EventStore\Exception\StorageConfigurationException;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class EventStoreManager
+{
+
+    /**
+     * @var array
+     * @Flow\InjectConfiguration(path="EventStore.stores")
+     */
+    protected $configuration;
+
+    /**
+     * @var array
+     */
+    protected $eventStoreIdentifiersPerBoundedContext = null;
+
+    /**
+     * @var array<EventStore>
+     */
+    protected $initializedEventStores;
+
+    public function getEventStoreForAggregateStreamName($streamName) {
+        // TODO
+        return $this->getEventStoreForEventTypes([$streamName]);
+    }
+
+    public function getEventStoreForEventTypes(array $eventTypes) : EventStore {
+        $boundedContexts = [];
+        foreach ($eventTypes as $eventName) {
+            list($boundedContext) = explode(':', $eventName);
+            $boundedContexts[$boundedContext] = $boundedContext;
+        }
+
+        $result = null;
+        foreach ($boundedContexts as $boundedContext) {
+            $tempResult = $this->getEventStoreForBoundedContext($boundedContext);
```

Comment:
`$result` and `$tempResult` are not the best variable names IMO ;)
Maybe ? `$eventStoreInstance` and `$previousEventStoreInstance` or sth?